packages/rainbowkit/src/components/ConnectOptions/MobileOptions.tsx

```diff
@@ -122,26 +181,28 @@ function WalletButton({
             width="60"
           />
         </Box>
-        <Box display="flex" flexDirection="column" textAlign="center">
-          <Text
-            as="h2"
-            color={wallet.ready ? 'modalText' : 'modalTextSecondary'}
-            size="13"
-            weight="medium"
-          >
-            {/* Fix button text clipping in Safari: https://stackoverflow.com/questions/41100273/overflowing-button-text-is-being-clipped-in-safari */}
-            <Box as="span" position="relative">
-              {shortName ?? name}
-              {!wallet.ready && ' (unsupported)'}
-            </Box>
-          </Text>
-
-          {wallet.recent && (
-            <Text color="accentColor" size="12" weight="medium">
-              {i18n.t('connect.recent')}
+        {!isCustomConnector ? (
+          <Box display="flex" flexDirection="column" textAlign="center">
+            <Text
+              as="h2"
+              color={wallet.ready ? 'modalText' : 'modalTextSecondary'}
+              size="13"
+              weight="medium"
+            >
+              {/* Fix button text clipping in Safari: https://stackoverflow.com/questions/41100273/overflowing-button-text-is-being-clipped-in-safari */}
+              <Box as="span" position="relative">
+                {shortName ?? name}
+                {!wallet.ready && ' (unsupported)'}
+              </Box>
```

Comment:
No need for this if we use custom connector since we will have a loading spinner instead on mobile.